.mcignore

```diff
@@ -0,0 +1,17 @@
+.git/
+node_modules/
+npm-debug.log
+.DS_Store
+*.sw[po]
+*.xpi
+*.pyc
+logs/
+dist/
+ping-centre/
+*.update.rdf
+bin/prerender.js
+bin/prerender.js.map
+/locales/
+.gitignore
+.eslintrc.js
+.eslintignore
```

Comment:
Actually, when we ignore all of `activity-stream/**`, we can copy over these eslint files now without breaking `./mach lint`